wp-content/themes/oceanwp-child/bbpress/content-single-topic.php

```diff
@@ -14,8 +14,104 @@
 
 <?php 
 $backgroundimg = get_the_post_thumbnail_url(get_the_ID(), 'full'); 
+$post_id = get_the_ID();
 ?>
 
+<style>
+	.page-header, #qt_bbp_reply_content_toolbar, .loop-item-0, .bbp-admin-links a:not(:last-child), #post-<?php echo $post_id; ?> {
+		display:none;
+	}
+
+	.content-area{
+		padding-right: 0px;
+    	border-right-width: 0px;
+	}
+
+	#main #content-wrap{
+		padding-top:0px;
+	}
+
+	.container, .container-sm, .container-md, .container-lg{
+		max-width:100%;
+		padding-right : 0px;
+		padding-left:0px
+	}
+
+	#bbpress-forums, #site-header-inner, #single-forum{
+		padding-right:100px;
+		padding-left:100px;
+	}
+
+	.wp-editor-container{
+		background: #EFF3F5;
+		border: 1px solid #AFC5CF !important;
+		box-sizing: border-box;
+		border-radius: 5px;
+	}
+
+	#bbp_reply_content {
+		font-size: 16px !important;
+		height: 77px;
+		min-height: 77px;
+	}
+
+	.bbp-topic-subheader{
+		background: linear-gradient(0deg, rgba(255 255 255 / 80%), rgba(255 255 255 / 80%)), url(<?php echo $backgroundimg; ?>); 
+		height:auto;
+		padding-top:100px;
+		padding-bottom:30px;
+		background-size:cover;
+	}
+
+	.bbp-author-avatar img{
+		border-radius :50%;
+		float : left !important;
+	}
+
+	.bbp-author-name{
+		display:inline !important;
+		font-weight:bold;
+		font-size: 16px;
+	}
+
+	.bbp-reply-author{
+		width:100% !important;
+		text-align :left !important;
+	}
+
+	.bbp-reply-post-date{
+		text-align :left !important;
+		display: block;
+		border-top :none !important;
+		border-bottom :none !important;
+		margin-left:92px
+	}
+
+</style>
+
+<div id="single-forum" class="bbp-topic-subheader">
+	<p class="links no-margin-bottom"><a href="../../../discuss">back to discussions</a></p>
```

Comment:
Can you use [site_url](https://developer.wordpress.org/reference/functions/get_site_url/) instead of relative links like this one?

Reply:
Have changed it.